week4/assignment/week4_assignment/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import {createBrowserRouter, RouterProvider} from "react-router-dom";
+import Login from "./pages/Login";
+import MyPage from "./pages/MyPage";
+import SignUp from "./pages/SignUp";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Login/>,
+  },
+  {
+    path: "/mypage",
+    element: <MyPage />,
+  },
+  {
+    path: "/signup",
+    element: <SignUp />,
+  },
+]);
+
+function App(){
+  return(
+    <>
+    <RouterProvider router={router}/>
+    </>
```

Comment:
```suggestion
    <RouterProvider router={router}/>
```

요기는 굳이 <></> 로 감싸주지 않아도 될 것 같고, prettier 설정이 제대로 안되어 있는 것으로 보이네요..!